.github/workflows/build.yml

```diff
@@ -1036,3 +1115,232 @@ jobs:
+
+      - name: Setup (arm64| ${{ contains(matrix.container, 'alpine') && 'musl' || 'glibc'  }})
+        if: matrix.arch == 'arm64' && !contains(matrix.container, 'alpine') && matrix.target_platform != 'darwin'
+        run: |
+          sudo apt-get update
+          sudo apt install -y gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
+
+      - name: Setup Musl
+        if: contains(matrix.container, 'alpine')
+        run: |
+          cd packages/profiling-node
+          curl -OL https://musl.cc/aarch64-linux-musl-cross.tgz
+          tar -xzvf aarch64-linux-musl-cross.tgz
+          $(pwd)/aarch64-linux-musl-cross/bin/aarch64-linux-musl-gcc --version
+
+      # configure node-gyp
+      - name: Configure node-gyp
+        if: matrix.arch != 'arm64'
+        run: |
+          cd packages/profiling-node
+          yarn build:bindings:configure
+
+      - name: Configure node-gyp (arm64, ${{ contains(matrix.container, 'alpine') && 'musl' || 'glibc'  }})
+        if: matrix.arch == 'arm64' && matrix.target_platform != 'darwin'
+        run: |
+          cd packages/profiling-node
+          yarn build:bindings:configure:arm64
+
+      - name: Configure node-gyp (arm64, darwin)
+        if: matrix.arch == 'arm64' && matrix.target_platform == 'darwin'
+        run: |
+          cd packages/profiling-node
+          yarn build:bindings:configure:arm64
+
+      # build bindings
+      - name: Build Bindings
+        if: matrix.arch != 'arm64'
+        run: |
+          yarn lerna run build:bindings --scope @sentry/profiling-node
+
+      - name: Build Bindings (arm64, ${{ contains(matrix.container, 'alpine') && 'musl' || 'glibc'  }})
+        if: matrix.arch == 'arm64' && contains(matrix.container, 'alpine') && matrix.target_platform != 'darwin'
+        run: |
+          cd packages/profiling-node
+          CC=$(pwd)/aarch64-linux-musl-cross/bin/aarch64-linux-musl-gcc \
+          CXX=$(pwd)/aarch64-linux-musl-cross/bin/aarch64-linux-musl-g++ \
+          BUILD_ARCH=arm64 \
+          yarn build:bindings
+
+      - name: Build Bindings (arm64, ${{ contains(matrix.container, 'alpine') && 'musl' || 'glibc'  }})
+        if: matrix.arch == 'arm64' && !contains(matrix.container, 'alpine') && matrix.target_platform != 'darwin'
+        run: |
+          cd packages/profiling-node
+          CC=aarch64-linux-gnu-gcc \
+          CXX=aarch64-linux-gnu-g++ \
+          BUILD_ARCH=arm64 \
+          yarn build:bindings:arm64
+
+      - name: Build Bindings (arm64, darwin)
+        if: matrix.arch == 'arm64' && matrix.target_platform == 'darwin'
+        run: |
+          cd packages/profiling-node
+          BUILD_PLATFORM=darwin \
+          BUILD_ARCH=arm64 \
+          yarn build:bindings:arm64
+
+      - name: Build Monorepo
+        if: steps.restore-build.outputs.cache-hit != 'true'
+        run: yarn build --ignore @sentry/serverless --ignore @sentry/browser --ignore @sentry/astro
```

Comment:
Can we avoid building entire monorepo here? And only build for `@sentry/profiling-node`?

Reply:
@AbhiPrasad let me try that. I know lerna has --include-dependencies, but I think it stops 1 at direct deps and we miss some.